### DISCUSSION TITLE
Issue with Self-Host Keygen Setup

### DISCUSSION THREAD

**Discussion by polasomeshwar** [keygen-sh/community#115] [Issue]
I am trying to setup self-host keygen , I have successfully Installed all the required tools (keygen, postgress, redis) , now when I tried to hit the API's for license creation it is failing (I have enabled PROXY too), I am able to hit only two API's(ping, health)

Can you please help me with this.

**Comment by ezekg** [comment]
Can you share an error? Most likely issue is that your `KEYGEN_HOST` environment variable is not configured correctly.

**Comment by polasomeshwar** [comment]
Yes adding KEYGEN_HOST solved the issue thank you for the solution.
